src/Integration.Vsix/InfoBar/InfoBarManager.cs

```diff
@@ -224,7 +241,7 @@ public void Close()
             #region IVsInfoBarUIEvents
             void IVsInfoBarUIEvents.OnActionItemClicked(IVsInfoBarUIElement infoBarUIElement, IVsInfoBarActionItem actionItem)
             {
-                this.ButtonClick?.Invoke(this, EventArgs.Empty);
+                this.ButtonClick?.Invoke(this, new InfoBarButtonClickedEventArgs(actionItem.Text));
```

Comment:
The only useful information is the button text
https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.interop.ivsinfobaractionitem?view=visualstudiosdk-2022